lib/Doctrine/ORM/Query/AST/Functions/AvgFunction.php

```diff
@@ -13,8 +13,7 @@
  */
 final class AvgFunction extends FunctionNode
 {
-    /** @var AggregateExpression */
-    private $aggregateExpression;
+    private AggregateExpression $aggregateExpression;
```

Comment:
Here I elected not to use `AggregateExpression|null` + assert inside `getSql`: although the property is not set in the constructor, it is always set in `parse()`. Otherwise, it can IMO stay uninitialized.